earthpy/tests/test_clip.py

```diff
@@ -100,19 +50,19 @@ def test_clip_poly():
 # TODO same test for points and lines
 
 
-def test_clip_multipoly():
+def test_clip_multipoly(poly_in_gdf, multi_gdf):
```

Comment:
here you are calling the fixtures. is that right?

Reply:
Yes! They act as function arguments, so that you can explicitly see what data are needed for each test.

Reply:
neat. this is a cool way to structure tests. i may have to redo some of the other tests for plotting based upon this now.